src/main/java/step2/domain/LottoNumber.java

```diff
@@ -0,0 +1,49 @@
+package step2.domain;
+
+import java.util.Objects;
+
+public class LottoNumber implements Comparable<LottoNumber> {
+
+    public static final int LOTTO_START_NUMBER_VALUE = 1;
+    public static final int LOTTO_LAST_NUMBER_VALUE = 45;
+
+    private int number;
```

Comment:
Value Object 구현 💯 
VO 구현으로 흩어져있던 상수와 로또 번호에 대한 검증 로직 등이 한곳에 모여 응집력이 높아졌네요 👍 
로또 번호가 한번 생성되고 나서 변경이 되지 않는다면, 해당 상태 값에 final 키워드를 추가하여
불변성을 유지하도록 변경하면 더 좋을 것 같아요!

https://codingnuri.com/seven-virtues-of-good-object/

```suggestion
public class LottoNumber implements Comparable<LottoNumber> {

    public static final int LOTTO_START_NUMBER_VALUE = 1;
    public static final int LOTTO_LAST_NUMBER_VALUE = 45;

    private final int number;
```